Source/WebCore/Modules/url-pattern/URLPatternCanonical.cpp

```diff
@@ -252,4 +257,30 @@ ExceptionOr<String> canonicalizeHash(const String& value, BaseURLStringType valu
     return dummyURL.fragmentIdentifier().toString();
 }
 
+ExceptionOr<String> callEncodingCallback(EncodingCallbackType urlComponent, String&& input)
```

Comment:
Unclear why input is a `String&&` given that the implementation doesn't `WTFMove()` it at all.

Reply:
Oops my bad! Thank you for catching this!

I'll refactor so that invoking the canonicalize function will have WTFMove in the parameters as well.